extension-base/src/main/java/com/azure/autorest/extension/base/model/codemodel/CodeModel.java

```diff
@@ -145,4 +145,13 @@ public TestModel getTestModel() {
     public void setTestModel(TestModel testModel) {
         this.testModel = testModel;
     }
+
+    // TODO (weidxu): side-car
+    private Object configuration;
+    public Object getConfiguration() {
+        return configuration;
+    }
+    public void setConfiguration(Object configuration) {
+        this.configuration = configuration;
+    }
```

Comment:
We probably not want it to say long (one or two months best). So just make it an `Object` and prepare to get rid of it when side-car comes.